docs/examples/driver_examples/Qcodes example with keithley 2450.ipynb

```diff
@@ -17,7 +17,7 @@
    "source": [
     "import qcodes as qc \n",
     "from qcodes.instrument_drivers.tektronix.Keithley_2450 import Keithley2450\n",
-    "from qcodes.dataset.database import initialise_database\n",
+    "from qcodes.dataset.sqlite.database import initialise_database\n",
```

Comment:
We should eventually import all these from `qcodes.dataset`

Reply:
i can do that now :)